docs/application/dotnet/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,607 @@
+        {
+            Log.Debug(LogTag, "OnPause Widget");
+            base.OnPause();
+        }
+        public override void OnDestroy(bool permanent)
+        {
+            Log.Debug(LogTag, "OnDestroy Widget");
+            base.OnDestroy(permanent);
+        }
+        public override void OnStop()
+        {
+            Log.Debug(LogTag, "OnStop Widget");
+            base.OnStop();
+        }
+    }
+
+    namespace CompBasedCsharp
+    {
+        internal class App : EFLComponentBasedApplication
+        {
+            internal static readonly string LogTag = "CompBasedCsharp";
+            App(IDictionary<Type, string> typeInfo) : base(typeInfo)
+            {
+            }
+
+            static void Main(string[] args)
+            {
+                Elementary.Initialize();
+                Elementary.ThemeOverlay();
+                Log.Error(LogTag, "start");
+                IDictionary<Type, string> typeInfo = new Dictionary<Type, string>();
+                typeInfo.Add(typeof(WidgetComp), "base-widget");
+                App app = new App(typeInfo);
+                app.Run(args);
+            }
+        }
+    }
+    ```
+
+<a name="launch_application"></a>
+## Launch Component Based Application
+
+To launch a component based application:
+
+1. Declare the launch privilege in the manifest file:
+    ```
+    <?xml version="1.0" encoding="utf-8"?>
+    <manifest xmlns="http://tizen.org/ns/packages" api-version="9" package="@PACKAGE_NAME@" install-location="internal-only" version="0.1.1">
+        <label>Sample</label>
+        <author email="tizenappfw@tizen.com" href="www.tizen.org">Tizen App Framework</author>
+        <description>Sample</description>
+        <ui-application appid="org.tizen.sample" exec="@BINDIR@/sample" nodisplay="false" multiple="false" type="dotnet" taskmanage="true" launch_mode="caller">
+            <label>Sample</label>
+            <icon>@DESKTOP_ICON@</icon>
+        </ui-application>
+        <privileges>
+            <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+        </privileges>
+    </manifest>
+    ```
+
+2.  Send launch request.
+
+    You can send a launch request using `AppControl`. `AppControl` contains the component based application's ID and the component ID. The component ID is optional. If you do not set the component ID, the main component instance will be created.
+
+    You can launch a component based application as shown in the following code:
+    ```
+    private void ButtonClickedCallback(object sender, EventArgs e)
+    {
+        AppControl control = new AppControl();
+        control.ApplicationId = "org.tizen.example.CompBasedCsharp";
+        control.ComponentId = "base-frame";
+        control.LaunchMode = AppControlLaunchMode.Group;
+        AppControl.SendLaunchRequestAsync(control, null);
+    }
+    ```
+
+
+## Related Information
+- Dependencies
+  - Tizen 5.5 or Higher for Mobile (Tizen 6.5 for Widget component)
```

Comment:
@annie-samsung It is not recommended. Instead, I recommend adding a note to the Widget Component description that it is supported since 6.5.

Reply:
> @annie-samsung It is not recommended. Instead, I recommend adding a note to the Widget Component description that it is supported since 6.5.

Hello, I applied your advice.
Could recheck them? Thank you!